### DISCUSSION TITLE
Change .output dir

### DISCUSSION THREAD

**Discussion by k0mar12** [nuxt/nuxt#18009] [Questions] [answered]
Is exist way to change default directory of `.output` like it exist for `.nuxt` ?

**Comment by danielroe** [comment, marked as answer]
Yes.
```js
import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
  nitro: {
    output: {
      dir: 'mydir'
    }
  }
})
```

**Reply by k0mar12** [reply]
Thank you :)

**Reply by k0mar12** [reply]
Oops, `defineNuxtConfig` hasnt declaration about `nitro`.  So i cant pass this param

**Reply by danielroe** [reply]
It should. If there are no types for the `nitro` key, please open an issue with a reproduction and I'll do my best to help.

**Reply by k0mar12** [reply]
@danielroe yes it is, my ide lying to me, im sorry, checked http://joxi.net/DmBJR67tgX5Wwm
